html/elements/input/password.json

```diff
@@ -0,0 +1,1070 @@
+                "deprecated": false
+              }
+            }
+          },
+          "autofocus": {
+            "__compat": {
+              "description": "<code>autofocus</code> attribute",
+              "support": {
+                "chrome": {
+                  "version_added": "5"
+                },
+                "chrome_android": {
+                  "version_added": "4"
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "4"
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": "10"
+                },
+                "opera": {
+                  "version_added": "10"
+                },
+                "opera_android": {
+                  "version_added": true
+                },
+                "safari": {
+                  "version_added": "5"
+                },
+                "safari_ios": {
+                  "version_added": null
+                },
+                "webview_android": {
+                  "version_added": null
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            }
+          },
+          "disabled": {
+            "__compat": {
+              "description": "<code>disabled</code> attribute",
+              "support": {
+                "chrome": {
+                  "version_added": "1"
+                },
+                "chrome_android": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "1",
+                  "notes": "Firefox will, unlike other browsers, by default, <a href='http://stackoverflow.com/q/5985839/432681'>persist the dynamic disabled state and (if applicable) dynamic checkedness</a> of an <code><input></code> across page loads. Setting the value of the <code><a href='https://developer.mozilla.org/docs/Web/HTML/Element/input#attr-autocomplete'>autocomplete</a></code> attribute to <code>off</code> disables this feature; this works even when the <code><a href='https://developer.mozilla.org/docs/Web/HTML/Element/input#attr-autocomplete'>autocomplete</a></code> attribute to <code>off</code> attribute would normally not apply to the <code><input></code> by virtue of its <code><a href='https://developer.mozilla.org/docs/Web/HTML/Element/input#attr-type'>type</a></code>. See <a href='https://bugzil.la/654072'>bug 654072</a>."
```

Comment:
This isn't at all specific to password inputs. It should be attached to the data for the generic input element.